flag_string.go

```diff
@@ -2,75 +2,30 @@ package cli
 
 import (
 	"flag"
-	"fmt"
 )
 
-// GetValue returns the flags value as string representation and an empty
-// string if the flag takes no value at all.
-func (f *StringFlag) GetValue() string {
-	return f.Value
-}
+// -- string Value
```

Comment:
Is there special significance to the `// --`?

Reply:
ha this is cut and paste from the golang flag code